client/src/lib/slate-pointers/PointerImportNode.tsx

```diff
@@ -258,7 +285,11 @@ class PointerImportNodePresentational extends React.Component<any, any> {
     const styles = StyleSheet.create({
       OuterPointerImportStyle: {
         ":before": {
-          backgroundColor: isLazyPointer
+          backgroundColor: isKnownAsMaliciousAnswerCandidate
+            ? "red"
+            : isKnownAsHonestAnswerCandidate
+            ? "green"
+            : isLazyPointer
```

Comment:
I see use of both `isLazyPointer` and `isLazy` for this same logic. Do you think we could use the same name or is there a good reason for them to be named differently?